digitalocean/resource_digitalocean_uptime_alert_test.go

```diff
@@ -0,0 +1,164 @@
+package digitalocean
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+const testAccCheckDigitalOceanUptimeAlertConfig_basic = `
+resource "digitalocean_uptime_check" "test" {
```

Comment:
```suggestion
data "digitalocean_account" "test" {
}

resource "digitalocean_uptime_check" "test" {
```

We need to add the data source referenced below.

Reply:
TY!